src/store/actions/api.actions.js

```diff
@@ -284,6 +303,9 @@ export const preparePlugins = api => api.plugins.map(plugin => {
 
         return R.set(lens2, policy.selected, updatedPlugin);
     }
+    if (plugin.name === 'oauth2') {
```

Comment:
I'd rather to use switch case here:

I'm under the impression I already reviewed this code 🤔